.github/workflows/tests.yml

```diff
@@ -8,53 +8,32 @@ on:
 
 name: Tests
 jobs:
-  test:
-    name: Test suite (mainnet, testnet, betanet)
+  test-mainnet:
+    strategy:
+      matrix:
+        network: [ mainnet, testnet ]
```

Comment:
Are you intentionally dropping the betanet?

Reply:
Yes.